civicrm.module

```diff
@@ -16,8 +16,8 @@ function civicrm_page_attachments(array &$page) {
     return;
   }
 
-  // Always add core styles (even if not on a /civicrm page).
```

Comment:
@jackrabbithanna @dsnopek I feel like these lines should stay in but the new L19 and L20 should be added thoughts?